File: __tests__/fixtures/output/php/http2/text-plain.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$body = new http\Message\Body;
$body->append('Hello World');

$request->setRequestUrl('https://httpbin.org/anything');
$request->setRequestMethod('POST');
$request->setBody($body);

$request->setHeaders([
  'content-type' => 'text/plain'
]);

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();

File: src/targets/java/asynchttp/fixtures/text-plain.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "https://httpbin.org/anything")
  .setHeader("content-type", "text/plain")
  .setBody("Hello World")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();